lockbox-ios/View/RootView.swift

```diff
@@ -46,16 +68,39 @@ class RootView: UIViewController, RootViewProtocol {
         self.presenter?.onViewReady()
     }
 
+    func showSidebar() {
```

Comment:
This seems fine for the current design case in which we only have one view that will be displayed in split screen; I wonder if it can be generified at all for any other future cases where we'd want to have the split screen show....

Reply:
Actually this is leftover code from a previous iteration that I wasn't happy with - it isn't called anymore. I'll just remove it :)